assets/constants.js

```diff
@@ -3,7 +3,7 @@ const ON_CHANGE_DEBOUNCE_TIMER = 300;
 const PUB_SUB_EVENTS = {
   cartUpdate: 'cart-update',
   quantityUpdate: 'quantity-update',
-  variantChangeStart: 'variant-change-start',
+  optionValueSelectionChange: 'option-value-selection-change',
```

Comment:
This event was updated to be focused on option value selection instead of variant selection. Because the theme only loads the variants associated with the [currently displayed option values](https://shopify.dev/docs/storefronts/themes/product-merchandising/variants/support-high-variant-products#option-value-availability), if a buyer clicks through option value selections very quickly, we do not know which variant is being switched to until the section rendering api request resolves.